### DISCUSSION TITLE
Use SolidPolygonLayer

### DISCUSSION THREAD

**Discussion by TQtong** [visgl/deck.gl#9108] [General]
I want to use SolidPolygonLayer api by source code, but it's not work,this is my change code.

<img width="285" alt="Snipaste_2024-08-27_09-04-26" src="https://github.com/user-attachments/assets/282a7783-e537-47b8-a507-5aaaf4b2451e">
<img width="917" alt="Snipaste_2024-08-27_09-03-11" src="https://github.com/user-attachments/assets/b758ba0a-d010-446c-8de2-76ccc9dafada">

this is problem

<img width="702" alt="Snipaste_2024-08-27_09-07-41" src="https://github.com/user-attachments/assets/8fbc37bd-dc52-44de-94bc-9d2973e44dc3">

**Comment by Pessimistress** [comment]
Do not change the import path, it does not work that way. Run `yarn start-local` in the app directory. https://github.com/visgl/deck.gl/blob/master/CONTRIBUTING.md#setting-up-dev-environment

**Reply by TQtong** [reply]
Thank you, let me try for it
